src/ngAria/aria.js

```diff
@@ -235,30 +251,24 @@ ngAriaModule.directive('ngShow', ['$aria', function($aria) {
           }
         },
         post: function(scope, elem, attr, ngModel) {
-          var needsTabIndex = shouldAttachAttr('tabindex', 'tabindex', elem)
-                                && !isNodeOneOf(elem, nodeBlackList);
+          var needsTabIndex = shouldAttachAttr('tabindex', 'tabindex', elem, false);
+          var ngTrueValue = $parse(attr.ngTrueValue)(scope);
 
           function ngAriaWatchModelValue() {
             return ngModel.$modelValue;
           }
 
           function getRadioReaction() {
-            if (needsTabIndex) {
-              needsTabIndex = false;
-              return function ngAriaRadioReaction(newVal) {
-                var boolVal = (attr.value == ngModel.$viewValue);
-                elem.attr('aria-checked', boolVal);
-                elem.attr('tabindex', 0 - !boolVal);
-              };
-            } else {
-              return function ngAriaRadioReaction(newVal) {
-                elem.attr('aria-checked', (attr.value == ngModel.$viewValue));
-              };
-            }
+            return function ngAriaRadioReaction(newVal) {
+              var boolVal = (attr.value == ngModel.$viewValue);
+              elem.attr('aria-checked', boolVal);
+            };
           }
-
-          function ngAriaCheckboxReaction() {
-            elem.attr('aria-checked', !ngModel.$isEmpty(ngModel.$viewValue));
+          function getCheckboxReaction() {
+            return function ngAriaCheckboxReaction(newVal) {
+              var modelValue = attr.ngTrueValue ? ngModel.$viewValue == ngTrueValue : newVal;
```

Comment:
The use case was the [Angular Material Checkbox](https://material.angularjs.org/latest/demo/checkbox), which uses `ng-true-value` on the second demo. But it still works when I revert this back to the way it was (while still excluding `INPUT`). Therefore I think we should revert this check and avoid going further into the weeds.